### PR TITLE
Show alert for WWD domains with a pending contact update.

### PR DESCRIPTION
**Pull request by gius80**
## Proposed Changes

This PR disables contact information editing for WWD domains that have a pending contact update. For these cases we are showing a specific message.

_Context: when a WWD domain with privacy off is updated, the user has to confirm the update (clicking on the link WWD sends to old and new email addresses). While the update is pending, additional contact info updates are disabled for 5 days (unless the user confirm the pending update)._
 
## Testing Instructions

This diff depends on D152677-code

If you don't have a WWD domain, you can force `has_pending_contact_update` property in the backend.

In the Contact Information card you should see an alert.

contact-info-pending

Note: this PR won't take into account bulk updates

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/domains/domain-management/settings/cards/contact-information/contacts-card.tsx [hunk 1]
```diff
@@ -218,6 +219,13 @@ const ContactsPrivacyCard = ( props: ContactsCardProps ) => {
 							) }
 						</p>
 					) }
+					{ pendingContactUpdate && (
+						<p className="contact-information__pending-update-warn">
+							{ translate(
+								"This domain has a pending contact information update. You will be able to update your contact information once the update is complete. If you don't confirm the update, the pending request will be canceled after 5 days."
```
Comment (leonardost) [inline client/my-sites/domains/domain-management/settings/cards/contact-information/contacts-card.tsx]:
```suggestion
								"This domain has a pending contact information update. You will be able to update your contact information once the pending update is complete. If you don't confirm the update, the pending request will be canceled after 5 days."
```
I think adding this makes the sentence a little clearer.

I wonder if we should also add something like "you should have received an email to confirm the contact information update" so users know where they can confirm that.

**Review by leonardost — APPROVED** [review]
Tested and LGTM!

File: client/my-sites/domains/domain-management/settings/cards/contact-information/contacts-card.tsx [hunk 2]
```diff
@@ -185,7 +186,7 @@ const ContactsPrivacyCard = ( props: ContactsCardProps ) => {
 					<ContactDisplay selectedDomainName={ selectedDomainName } />
 					<div className="contact-information__button-container">
 						<Button
-							disabled={ disableEdit || readOnly }
+							disabled={ disableEdit || readOnly || pendingContactUpdate }
 							href={
 								disableEdit || readOnly
```
Comment (leonardost) [inline client/my-sites/domains/domain-management/settings/cards/contact-information/contacts-card.tsx]:
```suggestion
								disableEdit || readOnly || pendingContactUpdate
```
I overlooked this before approving 😬 without this, even if the button has the disabled styling, it still works

Comment (gius80) [reply client/my-sites/domains/domain-management/settings/cards/contact-information/contacts-card.tsx]: Good catch!

**Pull request merged**